gstools/krige/base.py

```diff
@@ -9,23 +9,32 @@
 .. autosummary::
    Krige
 """
-# pylint: disable=C0103, W0221, E1102, R0201
+# pylint: disable=C0103, W0221, E1102, R0201, C0412
 import collections
 
 import numpy as np
 import scipy.linalg as spl
 from scipy.spatial.distance import cdist
 
+from gstools import config
 from gstools.field.base import Field
-from gstools.krige.krigesum import (
-    calc_field_krige,
-    calc_field_krige_and_variance,
-)
 from gstools.krige.tools import get_drift_functions, set_condition
 from gstools.tools.geometric import rotated_main_axes
 from gstools.tools.misc import eval_func
 from gstools.variogram import vario_estimate
 
+if config.USE_RUST:
+    # pylint: disable=E0401
+    from gstools_core import (  # pragma: no cover
+        calc_field_krige,
+        calc_field_krige_and_variance,
+    )
+else:
+    from gstools.krige.krigesum import (
+        calc_field_krige,
+        calc_field_krige_and_variance,
+    )
```

Comment:
What about setting `# pragma: no cover` behind `if config.USE_RUST:`?
We could also just add `E0401` in line 13.